tests/testthat/data/test_keeptimes.xml

```diff
@@ -0,0 +1,103 @@
+<wps:Execute service="WPS" version="1.0.0" xsi:schemaLocation="http://www.opengis.net/wps/1.0.0 http://schemas.opengis.net/wps/1.0.0/wpsExecute_request.xsd" xmlns:ogc="http://www.opengis.net/ogc" xmlns:ows="http://www.opengis.net/ows/1.1" xmlns:wps="http://www.opengis.net/wps/1.0.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
```

Comment:
This file was generated from the cida-based process WPS. It was being referenced by URL but that URL will no longer be active when the cida-based GDP is shut down.